Options.lua

```diff
@@ -2358,6 +2358,12 @@ local function CreateArenaWidgetOptions()
                 type = "toggle",
                 arg = {"arenaWidget", "Allies", "HideName"},
               },
+              UseFrameSort = {
+                name = L["Use FrameSort"],
+                order = 30,
+                type = "toggle",
+                arg = {"arenaWidget", "Allies", "UseFrameSort"},
+              },
```

Comment:
I would recommend to check here if FrameSort is actually installed and enabled. If not, print a error/warning message to chat. I do something like this for LibDogTag support, 

You can find an example in Options.lua, line 9878/9879. You can add the check for FrameSort either directoy or also add it to LoadOnDemandLibraries (which is also used during loading, not only when changing options). I guess it should be a new function like CheckForSupportedAddons or something like that.

Reply:
I've now hid the option when FrameSort is disabled.